SMA_FluxExtraction.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>Once this Gaussian information is calculated, it should be explained where to find the beam width it is compared against in order to make a final conclusion on the data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>